release/models/gnsi/openconfig-gnsi-acctz.yang

```diff
@@ -96,6 +102,20 @@ module openconfig-gnsi-acctz {
       }
     }
   }
+  typedef session-service {
+    description "enum SessionService.SessionServiceType";
```

Comment:
I don't see this in the acctz proto?

https://github.com/openconfig/gnsi/blob/a43096008ba06570eddbdd64531d775a9b25a4b3/acctz/acctz.proto#L114

Reply:
Fixed this